corehq/util/timer.py

```diff
@@ -175,3 +178,20 @@ def to_list(self, exclude_root=False):
 
 class TimerError(Exception):
     pass
+
+
+def time_method():
+    """Decorator to get timing information on a class method
+
+    The class must have a TimingContext instance as self.timing_context
+    """
+    def decorator(fn):
+        @wraps(fn)
+        def _inner(self, *args, **kwargs):
+            if self.timing_context.is_started():
+                with self.timing_context(fn.__name__):
```

Comment:
Is there a way args/kwargs could be included here? from your screenshot it looks like `create_profile` get's called 4 times (though cached). If that's a common occurrence for large apps, knowing the args will probably be very helpful. Could also make it opt-in by changing this decorator to something like `@time_method('module_id', 'form_id')`